pages/about.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link 
+  rel="stylesheet" 
+  href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link 
+  href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="../styles/about.css">
+
+  <title>About ATR</title>
+</head>
+<body>
+  <header class="header">
+    <div class="brand-logo">Brand</div>
+    <button type="button" aria-label="Mobile navigation toggle button" class="nav-toggle">
+      <span aria-hidden="true" class="hamburger"></span>
+    </button>
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-list-item"><a href="../index.html" class="nav-list-link link">Home</a></li>
+        <li class="nav-list-item"><a href="../index.html#program-section" class="nav-list-link link">Program</a></li>
+        <li class="nav-list-item"><a href="../index.html#join" class="nav-list-link link">Join</a></li>
+        <li class="nav-list-item"><a href="#partner-section" class="nav-list-link link">Partners</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">News</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">AT Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <!-- Headline -->
+    <section class="headline-section">
+      <h1 class="headline-title">Africa Tech Report</h1>
+      <p class="headline-description">
+        Over the past 18 months, the Africa Tech Report 
+        has implemented an ambitious umbrella initiative 
+        to connect the African continent to broadband by 2030 
+        and leverage digital tools for economic transformation, 
+        creating better jobs for more people.
+      </p>
+      <p class="contact-us">
+        Please contact us per Email for any further questions about The Africa Tech Report.
+        <a href="mailto:contact@atr.org" class="contact-mail">contact@atr.org</a>
+      </p>
+    </section>
+    <!-- Logo section -->
+    <section class="logo-section">
+      <h2 class="logo-section-title">The Africa Tech Report Logo</h2>
+      <p class="logo-description">
+        The logo of The Africa Tech Report was decided through a 
+        competition that took place from June 8th to July 7th of the last year.
+      </p>
+      <div class="big-logo">ATR</div>
+    </section>
+    <!-- Editions section -->
+    <section class="editions-section">
+      <h2 class="editions-section-title">See the past Africa Tech Reports</h2>
+      <p class="editions-description">
+        Take a look at the last two Africa Tech Report Editions.
+      </p>
+      <ul class="editions-list list">
+        <li class="editions-list-item">
```

Comment:
- [ ] Kindly place the elements in a horizontal row as in the design 👇 
- Original:
- 
![Screenshot (167)](https://user-images.githubusercontent.com/67087664/152137021-667885be-d85d-4657-b001-1c3d303b5b65.png)
- Yours:

![Screenshot (168)](https://user-images.githubusercontent.com/67087664/152136993-6313d62f-0aa0-4582-a10d-f34c1a81e387.png)